out_writeapi.go

```diff
@@ -418,12 +424,14 @@ func FLBPluginFlushCtx(ctx, data unsafe.Pointer, length C.int, tag *C.char) int
 		return output.FLB_ERROR
 	}
 
-	responseErr := checkResponses(ms_ctx, config.appendResults, false, &config.mutex)
+	_, responseErr := checkResponses(ms_ctx, config.appendResults, false, &config.mutex)
 	if responseErr != nil {
 		log.Printf("Checking append responses for output instance with id: %d failed in FLBPluginFlushCtx: %s", id, responseErr)
 		return output.FLB_ERROR
 	}
 
+	// Commenting to say that I believe this would be the ideal place to build a NewManagedStream
```

Comment:
You can use TODO here. Something like:

// TODO: build a newManagedStream if request count is above threshold

Reply:
Done.